projects/robot_arm/parser.py

```diff
@@ -2,7 +2,8 @@
 
 import torch.nn
 
-from typing import Callable, Literal
+from typing import Callable
+from typing_extensions import Literal
```

Comment:
The man himself had an answer to this issue 
![image](https://user-images.githubusercontent.com/641561/123716752-37bb8c80-d830-11eb-95d2-3d57cd810d88.png)